x-pack/plugins/apm/public/components/shared/transactions_table/index.tsx

```diff
@@ -194,7 +191,6 @@ export function TransactionsTable({
     mainStatisticsData: {
       transactionGroups,
       isAggregationAccurate,
```

Comment:
The names `showAggregationAccurateCallout` and `isAggregationAccurate` are not super descriptive. Perhaps change them like:

- `isAggregationAccurate` to `isMaxTransactionGroupsExceeded`
- `showAggregationAccurateCallout` to `showMaxTransactionGroupsExceededWarning`

Reply:
@achyutjhunjhunwala The logic for calculating `isAggregationAccurate` is correct. You should do the same for the service inventory page:

https://github.com/elastic/kibana/blob/5e7ee2cb5108408602286d9d8d2ff51e737426dd/x-pack/plugins/apm/server/routes/services/get_service_transaction_groups.ts#L149-L151